include/libp2p/transport/tcp/tcp_connection.hpp

```diff
@@ -97,6 +99,8 @@ namespace libp2p::transport {
 
     boost::system::error_code handle_errcode(
         const boost::system::error_code &e) noexcept;
+
+    friend class security::TlsAdaptor;
```

Comment:
uuu

Reply:
uu

Reply:
it is... need connected socket by reference w/o reorganizing all the stuff